src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.nav-links {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+.links-list {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.links-list__item {
+  margin: 0 10px;
+  position: relative;
+}
+
+.first-item {
+  margin-left: 0;
+}
+
+.last-item {
+  margin-right: 0;
+}
+
+.link {
+  text-transform: uppercase;
+  color: #000;
+  cursor: pointer;
+  line-height: 60px;
+  text-decoration: none;
+  display: inline-block;
+}
+
+.logo-img {
+  display: block;
+}
+
+.is-active {
+  color: var(--blue-color);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 37px;
+  height: 4px;
+  position: absolute;
+  bottom: 0;
+  background-color: var(--blue-color);
+  border-radius: 8px;
+}
+
+[data-qa]:hover {
+  color: var(--blue-color);
+  transition: all 0.3s;
```

Comment:
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x

you can read here about `transition all`